pkg/exporter/localmachine_exporter.go

```diff
@@ -0,0 +1,66 @@
+package exporter
+
+import (
+	"archive/zip"
+	"io"
+	"log"
+	"os"
+
+	"github.com/Azure/aks-periscope/pkg/interfaces"
+	"github.com/Azure/aks-periscope/pkg/utils"
+)
+
+// LocalMachineExporter defines an Local Machine Exporter
+type LocalMachineExporter struct{}
+
+var _ interfaces.Exporter = &LocalMachineExporter{}
+
+// Export implements the interface method
+func (exporter *LocalMachineExporter) Export(files []string) error {
+	hostName, err := utils.GetHostName()
+	if err != nil {
+		return err
+	}
+
+	outFile, err := os.Create(hostName + ".zip")
+
+	defer outFile.Close()
+
+	zipWriter := zip.NewWriter(outFile)
+	defer zipWriter.Close()
+
+	for _, file := range files {
+		log.Printf("File %s is being exported to local machine", file)
+		fileToZip, err := os.Open(file)
+		if err != nil {
+			return err
+		}
+		defer fileToZip.Close()
```

Comment:
Just a heads up, this `defer` will not be called until the end of the `func` so you will be opening a lot of file descriptors in the `range` function until you close them at the end. This could potentially be an issue with a large number of files

Reply:
This may be consistent with how others are doing it, in that case not an issue 😄

Reply:
Yes, I might leave this part as is for now, bc other classes seem to be using defer like this. If it does become an issue, I will fix it.